test/integration/M3/Base.t.sol

```diff
@@ -272,8 +295,8 @@ contract Base is Test, Utils {
         {
             StakingNodesManager.Init2 memory initParams = StakingNodesManager.Init2({
                 redemptionAssetsVault: ynETHRedemptionAssetsVaultInstance,
-                withdrawalManager: actors.ops.WITHDRAWAL_MANAGER,
-                stakingNodesWithdrawer: actors.ops.STAKING_NODES_WITHDRAWER
+                withdrawalManager: address(withdrawalsProcessor),
```

Comment:
we are only setting withdrawalsProcessor to do these actions since

running completeQueuedWithdrawals and processPrincipalWithdrawals are best bundled togetger